requirements.txt

```diff
@@ -1,7 +1,7 @@
 libyaz0>=0.5
```

Comment:
Just in general for this file, it looks like requirements.txt are allowed to have comments (general python comments # work). I think it would be helpful to section the requirements off based on what they are needed for (Building, disasm, diff, etc.).